### DISCUSSION TITLE
Does the GH actions job concurrency property solve "Merging Quickly" concerns?

### DISCUSSION THREAD

**Discussion by dcow** [intuit/auto#2363] [Q&A]
I'm reading up on `auto` to see if it would be a good fit for our team. There's a whole section right after *Getting Started* on *Merging Quickly* and the need to manually prevent multiple auto jobs from running at the same time. Does gh concurrency (https://docs.github.com/en/actions/using-jobs/using-concurrency) solve this? Essentially, you can configure jobs with a concurrency ID and jobs with the same ID are queued and run serially.

It seems like the whole *Merging Quickly* section might be able to be avoided/reworked.

**Comment by hipstersmoothie** [comment]
I think that would alleviate all concerns there. That could be a good note in the docs for GitHub action. Other CI should still be aware. Would appreciate a PR!
